src/router.js

```diff
@@ -19,7 +19,7 @@ import store from './store/index'
 Vue.use(VueRouter)
 
 const Router = new VueRouter({
-  mode: 'history',
```

Comment:
We'll get this in as soon as the backend is set to default to index for unrecognized urls